rules/rules.libsonnet

```diff
@@ -0,0 +1,23 @@
+{
+  prometheusRules+:: {
+    groups+: [
+      {
+        name: 'ceph.rules',
+        rules: [
+          {
+            record: 'ceph_node_down',
+            expr: |||
+              kube_node_status_condition{condition="Ready",job="kube-state-metrics",status="true"} * on (node) group_right() max(label_replace(ceph_disk_occupation,"node","$1","exported_instance","(.*)")) by (node) == 0
+            ||| % $._config,
+          },
+          {
+            record: 'ceph_aggregate_latency',
```

Comment:
rename to maybe `cluster:ceph_disk_latency:join_ceph_node_disk_irate1m` based on convention